src/Symlex/Router/TwigRouter.php

```diff
@@ -99,11 +99,12 @@ public function route($routePrefix = '', $servicePrefix = 'controller.web.', $se
             return $response;
         };
 
-        $indexRequestHandler = function (Request $request) use ($app, $container, $servicePrefix, $servicePostfix, $webRequestHandler) {
-            return $webRequestHandler('index', $request, 'index');
+        $indexRequestHandler = function (Request $request, $action='index') use ($app, $container, $servicePrefix, $servicePostfix, $webRequestHandler) {
+            return $webRequestHandler('index', $request, $action);
         };
 
         $app->get($routePrefix . '/', $indexRequestHandler);
+        $app->get($routePrefix . '/{action}', $indexRequestHandler)->assert('action', '.+');
```

Comment:
The idea of this router is, that you create a new controller for each "page" and the request is routed to the indexAction, if there is no action in the form of `/controller/action`. In your case, I would just copy and modify the existing router to match your requirements. It's just 100 lines of code, probably less than many project's routing configuration XML or YAML file.

If the pages are content pages from a CMS or file templates, I usually add a ContentController with actions for each language and query the page: For example, `/content/en/about` would be routed to `ContentController->enAction('about')`.